src/Console_TEST.cc

```diff
@@ -40,12 +40,12 @@ class Console_TEST : public ::testing::Test {
   /// \brief Clear out all the directories we produced during this test.
   public: virtual ~Console_TEST()
   {
-    EXPECT_TRUE(ignition::common::unsetenv(IGN_HOMEDIR));
+    EXPECT_TRUE(unsetenv(IGN_HOMEDIR));
```

Comment:
This is one of the places where removing the namespace is actually changing what function is called. I believe the tests are failing because this is calling [this](https://linux.die.net/man/3/unsetenv) instead of our cross-platform function.

Reply:
Hmm.. Let me add the following excludes then:
- `env`
- `unsetenv`
- `joinPaths`
- `removeAll`
- `uuid`
- `isDirectory`

This will result in just `ignition::` being removed, where applicable

Reply:
Tests pass locally for me